libbeat/reader/parser/parser.go

```diff
@@ -103,12 +106,16 @@ func NewConfig(pCfg CommonConfig, parsers []common.ConfigNamespace) (*Config, er
 			if err != nil {
 				return nil, fmt.Errorf("error while parsing container parser config: %+v", err)
 			}
+			if config.Stream != readjson.All {
+				suffix = config.Stream.String()
```

Comment:
What if suffix (the stream) is already set?

Reply:
Only one suffix is supported at the moment.

Reply:
Maybe we should error then.

Reply:
I have added an error.